### PR TITLE
fix: handle api

### PR DESCRIPTION
**Pull request by Duy-Tinh-Dev**
fix: handle api

### CODE REVIEW

**Comment by Duy-Tinh-Dev** [comment]
@lamxuanthinh @Giang-st1-00 help me checkkkkkk

File: apis/profile.ts
Comment (lamxuanthinh) [inline apis/profile.ts:1]: Please check type id user @DuyTinh2003

File: apis/video.ts
Comment (lamxuanthinh) [inline apis/video.ts:1]: Please setup type reponse for API video

File: apis/videoshort.ts
Comment (lamxuanthinh) [inline apis/videoshort.ts:1]: Please setup type reponse for API video short

File: components/common/ModalSelectCourse/Release/index.tsx
Comment (lamxuanthinh) [inline components/common/ModalSelectCourse/Release/index.tsx:1]: Please use message template is console.log("Error during (problem)": error)

File: components/common/ProfileHeader/index.tsx
Comment (lamxuanthinh) [inline components/common/ProfileHeader/index.tsx:1]: Please change log error

**Review by lamxuanthinh — APPROVED** [review]

**Comment by lamxuanthinh** [comment]
@DuyTinh2003  Please renovation your code

**Comment by Duy-Tinh-Dev** [comment]
@lamxuanthinh  help me check

**Comment by lamxuanthinh** [comment]
oki

**Pull request merged**